tensorflow/python/data/experimental/ops/readers.py

```diff
@@ -340,6 +340,41 @@ def make_csv_dataset_v2(
   tuple that corresponds to a batch of CSV rows. The features dictionary
   maps feature column names to `Tensor`s containing the corresponding
   feature data, and labels is a `Tensor` containing the batch's label data.
+  
+  Usage Example:
+  
+  Using IRIS dataset to show how to convert .csv file into a dataset.
+  
+  >>> train_dataset_url = "https://storage.googleapis.com/download.tensorflow.org/data/iris_training.csv"
+  >>> train_dataset_fp = tf.keras.utils.get_file(fname=os.path.basename(train_dataset_url), origin=train_dataset_url)
+  >>>
+  >>> column_names = ['sepal_length', 'sepal_width', 'petal_length', 'petal_width', 'species']
+  >>> feature_names = column_names[:-1]
+  >>> label_name = column_names[-1]
+  >>> 
+  >>> batch_size = 32
+  >>> train_dataset = tf.data.experimental.make_csv_dataset(
+  ...   train_dataset_fp,
+  ...   batch_size,
+  ...   column_names=column_names,
+  ...   label_name=label_name,
+  ...   num_epochs=1)
+  >>> features_batch, labels_batch = next(iter(train_dataset))
+  >>> print(features_batch)
+  <OrderedDict([('sepal_length', <tf.Tensor: shape=(32,), dtype=float32, numpy=
```

Comment:
@boronhub We don't want to delete the `id` field. Please go through this documentation: https://www.tensorflow.org/community/contribute/docs_ref
The correct syntax for values we are not sure about is `...`. 
As in the example inside the documentation, you can see `id=...`. Do the same thing here. For all four. Also, the order of values and which may come up is random. So do the same for those. 
That is, 
```
numpy=array([..., dtype=float32])
```
Make the same change for all four cases.

Reply:
+1